src/nimlsppkg/logger.nim

```diff
@@ -0,0 +1,22 @@
+import os,logging
+
+let rollingLog = newRollingFileLogger( getAppDir() / "nimlsp.log")
```

Comment:
Please stay consistent in spacing with the parenthesis.
And please put this log in the same folder it has been residing in to avoid confusion.